neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -97,6 +97,8 @@ public virtual Fixed8 NetworkFee
 
         public virtual Fixed8 SystemFee => Settings.Default.SystemFee.TryGetValue(Type, out Fixed8 fee) ? fee : Fixed8.Zero;
 
+        public readonly Fixed8 FeePerByte => NetworkFee / Size;
```

Comment:
@igormcoelho readonly isn't valid here; so this change you made doesn't build.
Also, I think this would end up doing the division every time the value is accessed, which is sub-optimal.